library/core/src/num/mod.rs

```diff
@@ -656,6 +657,31 @@ impl u8 {
     pub const fn is_ascii_control(&self) -> bool {
         matches!(*self, b'\0'..=b'\x1F' | b'\x7F')
     }
+
+    /// Returns an iterator that produces an escaped version of a `u8`,
+    /// as if it were an ASCII character.
+    ///
+    /// The behavior is identical to [`ascii::escape_default`][ascii::escape_default].
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(inherent_ascii_escape)]
```

Comment:
The doc examples in the standard library usually show the unstable features they're using:

```suggestion
    /// #![feature(inherent_ascii_escape)]
```